index.html.in

```diff
@@ -531,5 +531,9 @@
             <div class="lead text-warning well well-sm" data-text="Enable_JS_Warning"></div>
             <noscript><p><img src="@include_piwik_url@/piwik.php?idsite=@include_piwik_siteid@" style="border: 0;" alt=""></p></noscript>
         </div>
+        <div class="info-message well" style="margin: 8px; padding: 8px;">
```

Comment:
Um, should we hide it in mobile? Would ut take too much of screen space? ( leading to bad UX? )